gbfs-validator/nonSchemaValidation/check_geofencing_zones.js

```diff
@@ -0,0 +1,25 @@
+function checkGeofencingZones({ errors, data }) {
+  data.data?.geofencing_zones?.features?.map((feature) => {
+    const multypolygon = feature.geometry
+
+    for (const multi_coordinates of multypolygon.coordinates) {
+      for (const geo of multi_coordinates) {
```

Comment:
Following https://datatracker.ietf.org/doc/html/rfc7946#section-3.1.7 coordinates is "an array of Polygon coordinate arrays" so I propose to rename `geo` to `polygonCoordinates`